doc/api/crypto.md

```diff
@@ -1616,7 +1616,7 @@ Example:
 
 ```js
 const crypto = require('crypto');
-crypto.pbkdf2('secret', 'salt', 100000, 512, 'sha512', (err, derivedKey) => {
+crypto.pbkdf2('secret', 'salt', 100000, 64, 'sha512', (err, derivedKey) => {
   if (err) throw err;
   console.log(derivedKey.toString('hex'));  // '3745e48...aa39b34'
```

Comment:
You need to change this value as well (the first bytes should still be the same, but the last bytes will most likely be different).